komodo/release_transpiler.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+from argparse import ArgumentError, ArgumentParser, ArgumentTypeError
+from komodo import load_yaml, write_to_file, prettified_yaml
+
+rhel7_builtins = {
+    "python" : {
+        "rhel6": {
+            "py27" : "2.7.17",
+            "py36" : "3.6.9"
+        },
+        "rhel7": {
+            "py27" : "2.7.14+builtin",
+            "py36" : "3.6.9+builtin"
+        }
+    },
+    "qt5" : {
+        "rhel6": {
+            "py27" : None,
+            "py36" : "5.11.13",
+        },
+        "rhel7": {
+            "py27" : None,
+            "py36" : "5.11.13+builtin",
+        }
+    }
+}
+
+
+def build_matrix_file(release_base, release_folder, builtins):
+    py27 = load_yaml("{}/{}-py27.yml".format(release_folder, release_base))
+
+    py36 = load_yaml("{}/{}-py36.yml".format(release_folder, release_base))
+
+    all_packages = set(py36.keys()).union(py27.keys())
+
+    compiled = {}
+
+    for p in all_packages:
+        if p in builtins:
+            compiled[p] = builtins[p]
+        elif py27.get(p) == py36.get(p):
+            compiled[p] = py27.get(p)
+        else:
+            compiled[p] = {"py27": py27.get(p), "py36": py36.get(p)}
```

Comment:
Here, you'll get e.g.
```
dash:
  py27:
  py36: 1.12.0
dash-core-components:
  py27:
  py36: 1.10.0
dash-html-components:
  py27:
  py36: 1.0.3
dash-renderer:
  py27:
  py36: 1.4.1
dash-table:
  py27:
  py36: 4.7.0
```
This can be a bit confusing. We should document what this means in the `combine` and `transpiler` subparsers. 
And actually, maybe yaml `null` is more intuitive?

Reply:
documented format in argparse